egs/spgispeech/s5/RESULTS

```diff
@@ -0,0 +1,55 @@
+data/srilm/4gram.kn0223.gz  file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  0     zeroprobs,  logprob=  -9264101       ppl=  76.19736  ppl1=  91.24972
+data/srilm/4gram.kn0111.gz  file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  0     zeroprobs,  logprob=  -9268668       ppl=  76.36029  ppl1=  91.45296
+data/srilm/3gram.me.gz      file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  0     zeroprobs,  logprob=  -9318474       ppl=  78.16015  ppl1=  93.69934
+data/srilm/3gram.kn012.gz   file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  0     zeroprobs,  logprob=  -9408681       ppl=  81.52866  ppl1=  97.90927
+data/srilm/3gram.gt011.gz   file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  5187  zeroprobs,  logprob=  -9399744       ppl=  81.56597  ppl1=  97.97564
+data/srilm/3gram.kn011.gz   file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  0     zeroprobs,  logprob=  -9411733       ppl=  81.64512  ppl1=  98.05495
+data/srilm/3gram.gt012.gz   file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  5187  zeroprobs,  logprob=  -9410070       ppl=  81.96129  ppl1=  98.47032
+data/srilm/3gram.kn022.gz   file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  0     zeroprobs,  logprob=  -9435192       ppl=  82.54594  ppl1=  99.18209
+data/srilm/3gram.gt022.gz   file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  5187  zeroprobs,  logprob=  -9453120       ppl=  83.63024  ppl1=  100.5598
+data/srilm/3gram.kn023.gz   file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  0     zeroprobs,  logprob=  -9483614       ppl=  84.43693  ppl1=  101.5498
+data/srilm/3gram.gt023.gz   file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  5187  zeroprobs,  logprob=  -9494343       ppl=  85.26021  ppl1=  102.6022
+data/srilm/2gram.gt01.gz    file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  5187  zeroprobs,  logprob=  -1.031115e+07  ppl=  124.9826  ppl1=  152.8191
+data/srilm/2gram.me.gz      file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  0     zeroprobs,  logprob=  -1.033317e+07  ppl=  125.6362  ppl1=  153.6176
+data/srilm/2gram.kn01.gz    file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  0     zeroprobs,  logprob=  -1.033766e+07  ppl=  125.9001  ppl1=  153.9539
+data/srilm/2gram.kn02.gz    file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  0     zeroprobs,  logprob=  -1.036267e+07  ppl=  127.3817  ppl1=  155.8414
+data/srilm/2gram.gt02.gz    file  data/srilm/dev.txt:  196610  sentences,  4726024  words,  0  OOVs  5187  zeroprobs,  logprob=  -1.03542e+07   ppl=  127.5276  ppl1=  156.0618
+##
+#
+
+# Bare TDNN-F model
+#for x in exp/chain_cleaned/tdnn_1d_sp/decode_*; do grep WER $x/wer_* | utils/best_wer.sh ; done
```

Comment:
Minor comment: the model directories here are named `tdnn_1d_sp` but the script below is `tdnn_1a_sp`.